lighthouse-core/test/audits/viewport-test.js

```diff
@@ -24,7 +24,7 @@ describe('Mobile-friendly: viewport audit', () => {
   it('fails when no input present', () => {
     return assert.equal(Audit.audit({
       Viewport: -1
-    }).rawValue, false);
+    }).rawValue, -1);
```

Comment:
Sorry! One more thing can you add a test that checks that there's a `debugString` present for this case?